rpms/python-mongoengine/python-mongoengine.spec

```diff
@@ -0,0 +1,92 @@
+%define srcname mongoengine
+
+
+Name:           python-%{srcname}
+Version:        0.10.5
+Release:        2%{?dist}
+BuildArch:      noarch
+
+License:        MIT
+Group:          Development/Libraries
+Summary:        A Python Document-Object Mapper for working with MongoDB
+URL:            https://github.com/MongoEngine/mongoengine
+Source0:        https://github.com/MongoEngine/mongoengine/archive/v%{version}.tar.gz
+
+BuildRequires:  python-django
+BuildRequires:  python-pymongo
+BuildRequires:  python-pymongo-gridfs
+BuildRequires:  python-setuptools
+BuildRequires:  python-sphinx
+BuildRequires:  python2-devel
```

Comment:
The remaining dependencies are documented as being required to build the docs in the Fedora spec file.